### PR TITLE
[PT] Translation: basics/sigils

### PR DESCRIPTION
**Pull request by thiamsantos**
Update basics/sigils portuguese translation.

### CODE REVIEW

File: pt/lessons/basics/sigils.md [hunk 1]
```diff
@@ -131,6 +132,16 @@ iex> ~W/i love #{'e'}lixir school/
 ["i", "love", "\#{'e'}lixir", "school"]
 ```
 
+### NaiveDateTime
+
+Uma [NaiveDateTime](https://hexdocs.pm/elixir/NaiveDateTime.html) pode ser bem útil para criar rapidamente uam struct que representa um `DateTime` **sem** um timezone.
```
Comment (anderkonzen) [inline pt/lessons/basics/sigils.md]: Small typo in: rapidamente **uam** struct

Comment (thiamsantos) [reply pt/lessons/basics/sigils.md]: @anderkonzen fixed.

**Comment by anderkonzen** [comment]
Great, thanks!

**Comment by thiamsantos** [comment]
@anderkonzen I forgot to update the version number in the frontmatter.

**Pull request merged**